### PR TITLE
Class2-rockPaperScissors

### PR DESCRIPTION
**Pull request by cfissell50**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] All tests passed: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript: 30 points

**Checkpoint 2**
- [ ] The application works as it should: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript and the DOM API: 30 points

**Checkpoint 3**
- [ ] Use of React: 25 points
- [ ] Accesses an API: 25 points
- [ ] Proper use of documentation (commenting on code): 25 points
- [ ] The application functions as it should: 25 points

### CODE REVIEW

File: 01week/rockPaperScissors.js [hunk 1]
```diff
@@ -7,11 +7,18 @@ const rl = readline.createInterface({
...
+// written inside a whole function using if statements
+// rock beats scissors; paper beats rock; scissors beat paper
+// put the bulk in argument to minimize lines using pipes and ampersands
 function rockPaperScissors(hand1, hand2) {
-
-  // Write code here
-
+  if (hand1 === hand2){
+    return 'Tie!';
+  } else if (hand1 === 'rock' && hand2 === 'scissors' || hand1 === 'paper' && hand2 === 'rock' || hand1 === 'scissors' && hand2 === 'paper'){
```
Comment (reneemeyer) [inline 01week/rockPaperScissors.js]: You need to wrap each scenario in parentheses. Remember order of operations and PEDMAS. I.E `else if ( (hand1 === 'rock' && hand2 === 'scissors') || (hand1 === 'paper' && hand2 === 'rock') || .......`

**Review by reneemeyer — CHANGES_REQUESTED** [review]
Nice job, I like how you simplified the logic. What are some edge cases that you could guard against? For example, if hand 2 is 'dragon', your game would declare hand 2 the winner.

**Comment by reneemeyer** [comment]
@cfissell50 close this pull request and open one against YOUR gh-pages.

File: 01week/rockPaperScissors.js (hunk 1 above)
Comment (cfissell50) [reply 01week/rockPaperScissors.js]: I wrote another else if swapping out hand1 and hand 2 and replace the else with 'Someone is trying to cheat'

Also i wrapped the cases in parenthesis! Pull request to follow....

**Pull request closed without merging**